mappings/net/minecraft/entity/player/PlayerEntity.mapping

```diff
@@ -64,9 +77,14 @@ CLASS net/minecraft/class_1657 net/minecraft/entity/player/PlayerEntity
 	METHOD method_7258 isSpawnForced ()Z
 	METHOD method_7259 incrementStat (Lnet/minecraft/class_3445;)V
 		ARG 1 stat
+	METHOD method_7260 increaseRidingMotionStats (DDD)V
+		ARG 1 vx
+		ARG 3 vy
+		ARG 5 vz
```

Comment:
v?

Reply:
`velocityX/Y/Z` plz

Reply:
My choice was X/Y/Z imo. I don't see the need to specify that it's velocity, but @Pyrofab suggested this change.

Reply:
Does this method just increase the stat by sqrt(vx^2 + vy^2 + vz^2)? If yes, the parameters could be named `dx` `dy` `dz`, since they're just coordinate differences.

Reply:
yes use `dx/y/z` or `deltaX/Y/Z` if it's the case, just `d` preferred